mz_bokeh_package/utilities/graphql_api.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import annotations  # to support internal types as type hints
+
+import logging
+from functools import lru_cache
+from dataclasses import dataclass
+
+from jsonschema import validate, ValidationError
+from gql import Client, gql
+from gql.transport.requests import RequestsHTTPTransport, log as requests_logger
+
+from mz_bokeh_package import utilities
+
+requests_logger.setLevel(logging.WARNING)
+
+
+@dataclass
+class User:
+    id: str = ""
```

Comment:
both `id` and `name` are mandatory fields in the database, so they should never be empty. So here you should not set the default value of `""`.